addons/sms/models/ir_config_parameter.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import logging
+
+from odoo import api, models
+
+logger = logging.getLogger(__name__)
+
+class IrConfigParameter(models.Model):
+    _inherit = 'ir.config_parameter'
+
+    @api.model
+    def _neutralize(self):
+        super()._neutralize()
+
+        key = 'sms.endpoint'
+        if not self.get_param(key):
+            return
+        neutralized_value = 'https://iap-services-test.odoo.com'
+        param = self.set_param(key, neutralized_value)
+        logger.warning('Deactivated parameter `%s` from `%s` to `%s`', key, param, neutralized_value)
```

Comment:
The same code occurs for all endpoints.  Maybe we could factor it out?
```suggestion
    def _neutralize(self):
        super()._neutralize()
        self._neutralize_iap_service('sms.endpoint')

    # move this method somewhere else, at a generic place
    def _neutralize_iap_service(self, key):
        if self.get_param(key):
            neutralized_value = 'https://iap-services-test.odoo.com'
            param = self.set_param(key, neutralized_value)
            logger.info('Neutralized %r from %r to %r', key, param, neutralized_value)
```